frontend/src/components/PlayComponents/PlayerCard.js

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
 import styled from 'styled-components/macro';
+import { FaDollarSign, FaUser } from 'react-icons/fa';
 
 export default function PlayerCard({ player, active }) {
   return (
     <PlayerCardStyled active={active}>
-      <div>{player.name}</div>
-      <div>cash: {player.cash}</div>
-      <div>bet: {player.currentBet}</div>
-      <div>{active.toString()}</div>
+      <p>
+        <FaUser /> {player.name}
+      </p>
+      <p>
+        {' '}
```

Comment:
Whitespace. I see these so often in your code, that I start to think they are needed for the layout 🤔 if so use margin.